samples-generator/lib/Extensions.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+var Cesium = require('cesium');
+
+var defined = Cesium.defined;
+
+var Extensions = {};
+
+module.exports = Extensions;
```

Comment:
For consistency with other files, place `var Extensions = {};` below `module.exports = Extensions;`.